kie-server-parent/kie-server-services/kie-server-services-openshift/src/main/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepository.java

```diff
@@ -307,6 +373,12 @@ protected boolean isUpdateByKieServerProcessAllowed(DeploymentConfig dc, ConfigM
     private boolean isKieServerRuntime() {
         return System.getProperty(KieServerConstants.KIE_SERVER_ID) != null;
     }
+    
+    private boolean isKieServerGlobalDiscoveryEnabled() {
```

Comment:
Please put here explicit default value of global discovery, or at least a comment saying what is the default value, to improve readability of the code.

Reply:
Added  comment to indicate that false is the default value.